packages/blockchain/src/index.ts

```diff
@@ -271,12 +213,20 @@ export default class Blockchain implements BlockchainInterface {
     this.db = opts.db ? opts.db : level()
     this.dbManager = new DBManager(this.db, this._common)
 
+    if (this._common.consensusAlgorithm() === ConsensusAlgorithm.Casper) {
```

Comment:
maybe we can do a switch instead of if statements here? should be cleaner since we're just checking consensusAlgorithm()

Reply:
updated